src/reference/asciidoc/file.adoc

```diff
@@ -189,20 +189,26 @@ If this is a problem in your case you should call FileLocker.unlock(File file) y
 When filtering and locking files is not enough it might be needed to control the way files are listed entirely.
 To implement this type of requirement you can use an implementation of `DirectoryScanner`.
 This scanner allows you to determine entirely what files are listed each poll.
-This is also the interface that Spring Integration uses internally to wire FileListFilters FileLocker to the FileReadingMessageSource.
-A custom DirectoryScanner can be injected into the <int-file:inbound-channel-adapter/> on the `scanner` attribute.
+This is also the interface that Spring Integration uses internally to wire `FileListFilter` s and `FileLocker` to the `FileReadingMessageSource`.
+A custom `DirectoryScanner` can be injected into the `<int-file:inbound-channel-adapter/>` on the `scanner` attribute.
 
 [source,xml]
 ----
 <int-file:inbound-channel-adapter id="filesIn" directory="file:${input.directory}"
-    prevent-duplicates="true" scanner="customDirectoryScanner"/>
+     scanner="customDirectoryScanner"/>
 ----
 
 This gives you full freedom to choose the ordering, listing and locking strategies.
 
-IMPORTANT: It is important to understand that filters (including patterns, regex, prevent-duplicates etc) and lockers, are actually used by the scanner.
-Any of these attributes set on the adapter are subsequently injected into the scanner.
-For this reason, if you need to provide a custom scanner and you have multiple file inbound adapters in the same application context, each adapter must be provided with its own instance of the scanner, either by declaring separate beans, or declaring `scope="prototype"` on the scanner bean so that the context will create a new instance for each use.
+IMPORTANT: It is important to understand that filters (including `patterns`, `regex`, `prevent-duplicates` etc) and `lockers`, are actually used by the `scanner`.
+Any of these attributes set on the adapter are subsequently injected into the `scanner`.
+For this reason, if you need to provide a custom `scanner` and you have multiple file inbound adapters in the same application context, each adapter must be provided with its own instance of the scanner, either by declaring separate beans, or declaring `scope="prototype"` on the scanner bean so that the context will create a new instance for each use.
```

Comment:
Well, it depends on the `DirectoryScanner` implementation. For example our `DefaultDirectoryScanner` is stateless, so no reason to worry about sharing it between different adapters.
I found your  note as good catch.
Changing this paragraph to be consistent with existing state...